contracts/foundry.toml

```diff
@@ -7,10 +7,21 @@ libs = ["dependencies"]
 gas_reports = ["*"]
 optimizer = true
 optimizer_runs = 10_000
+remappings = [
+    "@openzeppelin/contracts/=dependencies/@openzeppelin-contracts-5.1.0/",
+    "@openzeppelin-contracts-upgradeable/=dependencies/@openzeppelin-contracts-upgradeable-5.1.0/",
+    "forge-std/=dependencies/forge-std-1.9.4/",
+]
+fs_permissions = [
+    { access = "read-write", path = "script/output/31337"},
+    { access = "read-write", path = "script/output/11155111"},
+    { access = "read-write", path = "script/output/241320161"}
+]
 
 [soldeer]
 recursive_deps = true
 
 [dependencies]
 forge-std = "1.9.4"
+"@openzeppelin-contracts-upgradeable" = "5.1.0"
```

Comment:
is there such a thing as dependabot for foundry?

Reply:
We're using `slither` for that purpose, whenever it finds a dependency with open CVE it raises and alert.